internal/auth/service.go

```diff
@@ -22,20 +28,48 @@ type Service interface {
 	UpdateUser(ctx context.Context, req domain.UpdateUserRequest) error
 	CreateHash(password string) (hash string, err error)
 	ComparePasswordAndHash(password string, hash string) (match bool, err error)
+	Get2FAStatus(ctx context.Context, username string) (bool, error)
+	Enable2FA(ctx context.Context, username string) (string, string, error)
+	Verify2FA(ctx context.Context, username string, code string) error
+	Verify2FALogin(ctx context.Context, username string, code string) error
+	Disable2FA(ctx context.Context, username string) error
 }
 
 type service struct {
 	log     zerolog.Logger
 	userSvc user.Service
 }
 
+var (
+	ntpServer = "pool.ntp.org"
```

Comment:
```suggestion
	ntpServer = "0.beevik-ntp.pool.ntp.org"
```

Reply:
Source: https://github.com/beevik/ntp?tab=readme-ov-file#querying-the-current-time